packages/grant-explorer/src/features/discovery/ExploreProjectsPage.tsx

```diff
@@ -103,6 +107,17 @@ export function ExploreProjectsPage(): JSX.Element {
 
   const { projects, add, remove } = useCartStorage();
 
+  useEffect(() => {
```

Comment:
you don't need the useEffect and the useState here, you can compute this on render or in a useMemo

https://react.dev/learn/you-might-not-need-an-effect

Reply:
+1

Reply:
Thanks. Fixed.